src/js/components/TaskTable.js

```diff
@@ -165,6 +184,12 @@ class TaskTable extends React.Component {
     );
   }
 
+  renderHost(prop, task) {
+    let marathonTask = MarathonStore.getTaskFromTaskID(task.id);
```

Comment:
Good point, but what if the ports or hosts change for a given task? Then this component won't render the updated data because the task ID won't change.